akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/BasicPersistentBehaviorTest.java

```diff
@@ -355,7 +357,7 @@ public Recovery recovery() {
       // #tagging
       @Override
       public Set<String> tagsFor(Event event) {
-        throw new RuntimeException("TODO: inspect the event and return any tags it should have");
+        return Set.of("tag1", "tag2");
```

Comment:
this is pretty much what the scala docs are showing

Reply:
Oh, Set.of(), introduced in Java 9...

Reply:
We release for Java 8 compatibility and we have Jenkins jobs for verifying, so I think this will not compile in those.

Reply:
Yes, but I was surprised that the travis job that seems to run in jdk11 also failed. 
Anyway I may have missed something. 

I will use old-style Set construction. 😞